src/Orleans.EventSourcing/LogStorage/LogConsistencyProvider.cs

```diff
@@ -31,31 +24,6 @@ public bool UsesStorageProvider
             }
         }
 
-        /// <summary>
-        /// Init method
-        /// </summary>
-        /// <param name="name">Consistency provider name</param>
-        /// <param name="providerRuntime">Provider runtime</param>
-        /// <param name="config">Provider config</param>
-        public Task Init(string name, IProviderRuntime providerRuntime, IProviderConfiguration config)
-        {
-            Name = name;
-            var loggerName = $"{this.GetType().FullName}.{Name}";
-            var loggerFactory = providerRuntime.ServiceProvider.GetRequiredService<ILoggerFactory>();
-            this.logger = loggerFactory.CreateLogger(loggerName);
-            logger.Info("Init");
```

Comment:
add the logging back and make it log out something  more meaningful

Reply:
Why? This was only there because it had to have these calls because it was a provider.  I'm not opposed to improving the logging for this system, but that's not the goal of this PR.  Removing this cruft is a good thing.